src/infrastructure/event/CreateRequestEvent.ts

```diff
@@ -0,0 +1,12 @@
+class CreateRequestEvent {
+
+  // @ts-ignore
+  static name() {
```

Comment:
Is there any TS feature that would allow us to drop these `name()` static methods in event classes?

Reply:
Not recommended. When you want to use DI lib (e.g. inversify) you will need that fields for reflection in TS